<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1126</width>
    <height>847</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Tongue Tracking System v1.0</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>120</y>
      <width>941</width>
      <height>37</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="utteranceInfoLay">
     <item>
      <widget class="QComboBox" name="classBox">
       <property name="font">
        <font>
         <pointsize>18</pointsize>
        </font>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="utteranceBox">
       <property name="font">
        <font>
         <pointsize>18</pointsize>
        </font>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="trialBox">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <pointsize>18</pointsize>
        </font>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="measureEMFButton">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>270</y>
      <width>201</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Measure EMF</string>
    </property>
    <property name="shortcut">
     <string>Return</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="utteranceBrowser_disp">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>190</y>
      <width>221</width>
      <height>51</height>
     </rect>
    </property>
    <property name="html">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:8.25pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:24pt; font-weight:600;&quot;&gt;Utterance:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="utteranceBrowser">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>190</y>
      <width>601</width>
      <height>51</height>
     </rect>
    </property>
    <property name="html">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:8.25pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>350</y>
      <width>50</width>
      <height>110</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="volumeHLay">
     <item>
      <widget class="QProgressBar" name="leftVolumeBar">
       <property name="value">
        <number>24</number>
       </property>
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QProgressBar" name="rightVolumeBar">
       <property name="value">
        <number>24</number>
       </property>
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>370</y>
      <width>164</width>
      <height>161</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="videoLay" stretch="3,1">
     <item>
      <widget class="QLabel" name="video_label">
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <property name="styleSheet">
        <string notr="true">Background-color: #000;</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="showVideoCheckBox">
       <property name="font">
        <font>
         <pointsize>20</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Show Video</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="startStopTrialButton">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>270</y>
      <width>301</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Start</string>
    </property>
    <property name="shortcut">
     <string>Space</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QGroupBox" name="configBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>1081</width>
      <height>91</height>
     </rect>
    </property>
    <property name="title">
     <string>Configuration</string>
    </property>
    <widget class="QLabel" name="magnetLabel">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>60</y>
       <width>55</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Magnet</string>
     </property>
     <property name="buddy">
      <cstring>ttsSystEdit</cstring>
     </property>
    </widget>
    <widget class="QLineEdit" name="magnetEdit">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>60</y>
       <width>133</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="configButton">
     <property name="geometry">
      <rect>
       <x>730</x>
       <y>20</y>
       <width>131</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>Config Done</string>
     </property>
    </widget>
    <widget class="QLabel" name="subNbLabel">
     <property name="geometry">
      <rect>
       <x>21</x>
       <y>31</y>
       <width>44</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Subject#</string>
     </property>
     <property name="buddy">
      <cstring>subNbEdit</cstring>
     </property>
    </widget>
    <widget class="QLabel" name="expFileLabel">
     <property name="geometry">
      <rect>
       <x>471</x>
       <y>30</y>
       <width>73</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Experiment File</string>
     </property>
     <property name="buddy">
      <cstring>expFileEdit</cstring>
     </property>
    </widget>
    <widget class="QLineEdit" name="subNbEdit">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>31</y>
       <width>133</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="expFileEdit">
     <property name="geometry">
      <rect>
       <x>550</x>
       <y>30</y>
       <width>133</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="ttsSystEdit">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>30</y>
       <width>133</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="ttsSystLabel">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>30</y>
       <width>55</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>TTS system</string>
     </property>
     <property name="buddy">
      <cstring>ttsSystEdit</cstring>
     </property>
    </widget>
    <widget class="QLineEdit" name="subPathEdit">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>57</y>
       <width>133</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="subPathLabel">
     <property name="geometry">
      <rect>
       <x>21</x>
       <y>57</y>
       <width>61</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Subject Path</string>
     </property>
     <property name="buddy">
      <cstring>subPathEdit</cstring>
     </property>
    </widget>
   </widget>
   <widget class="QCameraViewfinder" name="cameraPlayer" native="true">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>370</y>
      <width>301</width>
      <height>171</height>
     </rect>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QCameraViewfinder</class>
   <extends>QWidget</extends>
   <header>qcameraviewfinder.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
